.projenrc.js

```diff
@@ -22,6 +22,13 @@ const project = new awscdk.AwsCdkConstructLibrary({
   ],
   npmAccess: NpmAccess.PUBLIC,
   releaseToNpm: true,
+  publishToPypi: {
+    distName: 'cdk-vpc-module',
+    module: 'cdk_vpc_module',
+  },
+  publishToGo: {
+    moduleName: 'cdk-vpc-module',
```

Comment:
```suggestion
    moduleName: 'cdk-vpc-module-go',
```

it has to be a name of a new repo.